hw1/02-palindrome.html

```diff
@@ -6,20 +6,31 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
+  <body class="bg-primary">
     <main>
       <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
-        </div>
-        <div>
-          <p>Is this a palindrome?</p>
-          <div></div>
+        <div class="container mt-5">
+          <div class="row justify-content-center">
+            <div class="col-sm-5 bg-light p-3 rounded">
```

Comment:
Palindrome card could be longer horizontally at small and medium sizes: just a preference since the homework doesn't specify.

<img width="966" alt="Screenshot 2023-04-14 at 1 53 50 PM" src="https://user-images.githubusercontent.com/66976595/232152680-a4c1599e-3699-4a23-816d-c906ab1a42b5.png">

Reply:
fixed